src/DynamoCore/Graph/Workspaces/SerializationConverters.cs

```diff
@@ -181,11 +181,13 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
             else if (type == typeof(Function))
             {
                 var functionId = Guid.Parse(obj["FunctionSignature"].Value<string>());
+                var DisplayName = obj["Description"].Value<string>();
```

Comment:
I don't think this is a valid fix.

The description field of a custom node will not always contain the name of the custom node - a user can put any description they want in here. Try creating a new custom node and adding an arbitrary description...

Reply:
At least this is the expected default behavior by looking at code, or you think this is why we do not deserialize description in the first place? Then seems a tech debt for our team to address regardless of this bug because we are losing info when reading the file.. If you talk about arbitrary info, that applies to regular nodes as well, how do we justify those cases?

Reply:
hmm, can you point me to where the description of a custom node is serialized?

Reply:
If you create a new custom node with some arbitrary  description, that will be serialized into dyf but not the original dyn correct?